### PR TITLE
Add next steps after signing up in Check-in

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

---

**Related issue :**

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
Hi!

I have a question when reading https://docs.egi.eu/users/check-in/joining-virtual-organisation/

Should PERUN VOs be searchable in the "Enrollment Flows" step?

Best regards,
Sebastian

**Comment by gwarf** [comment]
@vardizzo-lab , @NicolasLiampotis , @ioigoume can you please check this PR?

File: content/en/users/check-in/signup/_index.md [hunk 1]
```diff
@@ -104,3 +105,9 @@ Information about identites linked to your account. Check the
 [guide for linking accounts](../linking) for more information.
 
 ![Linked identities](./check-in-profile-linked.png)
+
+## Next steps
+
+Once your Check-in account is ready you can check
+[how to link it with different identities](../linking) or
+[how to join an existing Virtual Organisation (VOs)](../joining-virtual-organisation).
```
Comment (NicolasLiampotis) [inline content/en/users/check-in/signup/_index.md:113]: "how to join an existing Virtual Organisation (VO)" (without the s)

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Minor typo in the next steps section (VO vs VOs) - other than that LGTM - thanks!

**Comment by gwarf** [comment]
Thanks for the review @NicolasLiampotis 
@vardizzo-lab or @NicolasLiampotis can you also please have a look at @sebastian-luna-valero 's question in https://github.com/EGI-Federation/documentation/pull/365#issuecomment-971545259

Let me know when we can merge this PR.

**Comment by NicolasLiampotis** [comment]
> I have a question when reading https://docs.egi.eu/users/check-in/joining-virtual-organisation/
> 
> Should PERUN VOs be searchable in the "Enrollment Flows" step?
> 

Currently the "Enrollment flows" list only includes the VOs that are managed in COmanage. We can investigate with the Ops Portal / Perun teams how to include Perun-managed VOs

**Comment by gwarf** [comment]
OK, I've logged a ticket about this in the Jira project for EGI Requirements so that it's assessed/discussed properly.
@NicolasLiampotis can you please mark this PR as approved if it's fine with you?

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**